lib/api/funnel.js

```diff
@@ -269,6 +269,10 @@ class Funnel {
             })
             .then(allowed => {
               if (!allowed) {
+                const { controller, action } = _request.input;
+                if (controller === 'auth' && action === 'login') {
```

Comment:
Missing unit tests for this case

Reply:
@sagarmulchandani you can duplicate this unit test and change the `request.input.controller` and `request.input.action` property to simulate a `auth:login` request:
https://github.com/kuzzleio/kuzzle/blob/master/test/api/funnel/execute.test.js#L132

Reply:
Hi @Aschen , have made the changes but facing an issue with running `npm test` and have been occupied a little with my office work to look into this. Will be fixing this and updating soon. Apologies for the delay on my end.

Reply:
You can join us on [Discord](http://join.discord.kuzzle.io) if you need some help with the tests